### PR TITLE
Bug 1558532 - build windows images with worker-runner

### PR DESCRIPTION
**Pull request by djmitche**
Work in progress to build windows images that run worker-runner.

### CODE REVIEW

**Comment by djmitche** [comment]
Note that I had to remove some out-commented code, and some comments, from this file to keep it under 16k.

**Comment by djmitche** [comment]
@petemoore I've tested this with the win2016 image, but haven't yet built the other images as I want to see if the approach is OK before I go to all that effort.

File: imagesets/firefoxreality-1-win10-64/bootstrap.ps1 [hunk 1]
```diff
@@ -18,181 +11,124 @@ function Expand-ZIPFile($file, $destination, $url)
...
-# allow powershell scripts to run
 Set-ExecutionPolicy Unrestricted -Force -Scope Process
-
-# install chocolatey package manager
 Invoke-Expression ($client.DownloadString('https://chocolatey.org/install.ps1'))
-
-# install Windows SDK 8.1
 choco install -y windows-sdk-8.1
-
-# install June 2010 DirectX SDK for compatibility with Win XP
```
Comment (petemoore) [inline imagesets/firefoxreality-1-win10-64/bootstrap.ps1]: I think these comments provide value, for example identifying that this SDK is useful if builds on the worker need to provide compatibility with Win XP.

File: imagesets/firefoxreality-1-win10-64/bootstrap.ps1 [hunk 2]
```diff
@@ -18,181 +11,124 @@ function Expand-ZIPFile($file, $destination, $url)
...
-
-# now run DirectX SDK installer
 Start-Process C:\DXSDK_Jun10.exe -ArgumentList "/U" -Wait -NoNewWindow -PassThru -RedirectStandardOutput C:\directx_sdk_install.log -RedirectStandardError C:\directx_sdk_install.err
-
-# install rustc dependencies (32 bit)
 $client.DownloadFile("http://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x86.exe", "C:\vcredist_x86-vs2013.exe")
 Start-Process "C:\vcredist_x86-vs2013.exe" -ArgumentList "/install /passive /norestart /log C:\vcredist_x86-vs2013-install.log" -Wait -PassThru
-
-# install rustc dependencies (64 bit)
-$client.DownloadFile("http://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x64.exe", "C:\vcredist_x64-vs2013.exe")
```
Comment (petemoore) [inline imagesets/firefoxreality-1-win10-64/bootstrap.ps1]: Also here, we lose the information here that this is a `rustc` dependency if we delete the comment.

File: imagesets/firefoxreality-1-win10-64/bootstrap.ps1 [hunk 3]
```diff
@@ -18,181 +11,124 @@ function Expand-ZIPFile($file, $destination, $url)
...
-# complete package list: https://cygwin.com/packages/package_list.html
 Start-Process "C:\cygwin-setup-x86_64.exe" -ArgumentList "--quiet-mode --wait --root C:\cygwin --site http://cygwin.mirror.constant.com --packages openssh,vim,curl,tar,wget,zip,unzip,diffutils,bzr" -Wait -NoNewWindow -PassThru -RedirectStandardOutput "C:\cygwin_install.log" -RedirectStandardError "C:\cygwin_install.err"
-
-# open up firewall for ssh daemon
 New-NetFirewallRule -DisplayName "Allow SSH inbound" -Direction Inbound -LocalPort 22 -Protocol TCP -Action Allow
-
-# workaround for https://www.cygwin.com/ml/cygwin/2015-10/msg00036.html
-# see:
-#   1) https://www.cygwin.com/ml/cygwin/2015-10/msg00038.html
-#   2) https://cygwin.com/git/gitweb.cgi?p=cygwin-csih.git;a=blob;f=cygwin-service-installation-helper.sh;h=10ab4fb6d47803c9ffabdde51923fc2c3f0496bb;hb=7ca191bebb52ae414bb2a2e37ef22d94f2658dc7#l2884
```
Comment (petemoore) [inline imagesets/firefoxreality-1-win10-64/bootstrap.ps1]: Again, this is a very useful comment.

**Pull request merged**